### ISSUE TITLE
The theme is alive!

### ISSUE THREAD

**Issue by marcissimus** [sayontan/suffusion#50]
Hi Everyone, 
@mbrsolution, @Colinsp, @RogerKIWI, @vriesm, @fastasleep, @Trip69, @georgyo, @Lzr0, @Ciprian-Dracea

@sayontan was so kind to add me as a collaborator to this repo. I'm still running Suffusion on a few WP sites. There were a few things to clean up and adjust for newer WP and PHP releases. But now it's up-to-date and I will upload a new release to the `dist` folder soon. 

FYI, I'm not planning on any functional enhancements to the theme. But I'm happy to correct things if necessary. Feel free to create issues or pull requests!

Best,
Marc

**Comment by mbrsolution** [comment]
Thank you @marcissimus for keeping the theme alive :) I have been using Suffusion for many years and I don't really want to change themes. Changing themes can become a headache :(. I think it would be great to somehow organize the translation folder. I run a Spanish blog and have a full translation PO file as well.

Anyhow well done for keeping the theme alive.

Kind regards,
Manuel

**Comment by fastasleep** [comment]
Yes, thank you @marcissimus !  I have a single client using Suffusion who runs a very busy music blog for quite some time now, and I have no interest in migrating him to a new theme. These updates are a godsend. Gonna update to the new version shortly. :)

**Comment by RogerKIWI** [comment]
Many thanks @marcissimus for this. I use it on my personal site, and have no interest in having to choose and modify a new theme, so I am very grateful to you for keeping this one alive.

Roger

**Comment by mbtools** [comment]
Great to see that I'm not the last using the theme 😉 

Regarding translation @mbrsolution, you can create a pull request here to add `es_ES.po/mo` to the [translation folder](https://github.com/sayontan/suffusion/tree/master/code/translation). Or attach the file here and I will take care of it.

**Comment by marcissimus** [comment]
Great to see that I'm not the last using the theme 😉

Regarding translation @mbrsolution, you can create a pull request here to add es_ES.po/mo to the [translation folder](https://github.com/sayontan/suffusion/tree/master/code/translation). Or attach the file here and I will take care of it.

**Comment by mbrsolution** [comment]
@marcissimus I see that you have created a POT file. This is great news. I had to recheck my original translation. Here is the Spanish translation files po/mo zipped for you :)
[suffusion-es.zip](https://github.com/sayontan/suffusion/files/13454392/suffusion-es.zip)

Regards.

**Comment by marcissimus** [comment]
Great. I added the Spanish files here. Let me know if this works.

I also created a project on https://crowdin.com/project/suffusion. Send me an email and I will add you.

**Comment by mbrsolution** [comment]
Thank you Marc, I tested the translations file and it works. Just one question. Will you be adding the languages into the 4.6.0 compressed folder or will the person who wants to use the language have to download it from here? 

Also, do you think it is better to add suffusion.pot file instead of suffusion.po file into the 4.6.0 compressed folder? I also sent you an email so that you can add me to https://crowdin.com/project/suffusion.

Regards,
Manuel

**Comment by mbtools** [comment]
I will add the translations and pot to 4.6.1

**Comment by marcissimus** [comment]
https://github.com/sayontan/suffusion/releases/tag/v4.6.1

I think that's it for a while now. Enjoy!

**Comment by mbrsolution** [comment]
Thank you @marcissimus, for the release of version 4.6.5. It is much appreciated :) Thank you also for sharing the screen capture for the Query Posts widget. That helped me to reproduce the warning message to versions prior to 4.6.5. After installing 4.6.5 everything is working as expected. Well done.

Kind regards,
Manuel

**Comment by mrai64** [comment]
_Gooodmooooooorniiiing Maaaricissimus!_
I need to adapt Suffusion to actual wp 6.8.2 with php 8.4 under the hood,
i wanna install MAMP with wp and Suffusion theme in localhost,
then solve one issue at time. Or are all fine with 4.6.5 release?

**Comment by marcissimus** [comment]
Continue discussion in #103
